zerver/decorator.py

```diff
@@ -929,14 +929,15 @@ def internal_api_view(
 
     def _wrapped_view_func(
         view_func: Callable[Concatenate[HttpRequest, ParamT], HttpResponse],
-    ) -> Callable[Concatenate[HttpRequest, ParamT], HttpResponse]:
+    ) -> Callable[..., HttpResponse]:
```

Comment:
Why this change?

Reply:
Looks like it's this: https://chat.zulip.org/#narrow/stream/49-development-help/topic/mypy.20exception/near/1927024

Seems fine to merge this for now.